src/main/fc/fc_init.c

```diff
@@ -478,8 +478,9 @@ void init(void)
     initBoardAlignment(boardAlignment());
 
     if (!sensorsAutodetect()) {
-        // if gyro was not detected due to whatever reason, we give up now.
-        failureMode(FAILURE_MISSING_ACC);
+        // if gyro was not detected due to whatever reason, notify and don't arm.
+        failureLedCode(FAILURE_MISSING_ACC, 2);
```

Comment:
Do we really want to treat this condition as special, compared to all the other conditions that disable arming? I think they should be indicated in a consistent way, and not all implement their own user signaling.